features/buildflags.prf

```diff
@@ -122,7 +122,7 @@ win32 {
 unix {
     !macx* {
         CONFIG(debug, debug|release):QMAKE_CXXFLAGS_DEBUG += -ggdb
-        QMAKE_CXX = colorgcc
+		QMAKE_CXXFLAGS += -Wno-unused-local-typedefs
```

Comment:
Almost the entire repo uses spaces instead of tabs. Not that I'm not open to change but we're not going to use both. See url for illustration ;)

http://www.emacswiki.org/pics/static/TabsSpacesBoth.png